storage/cache/subtree_cache.go

```diff
@@ -373,7 +355,7 @@ func (s *SubtreeCache) Flush(ctx context.Context, setSubtrees SetSubtreesFunc) e
 
 // newEmptySubtree creates an empty subtree for the passed-in ID.
 func (s *SubtreeCache) newEmptySubtree(id []byte) *storagepb.SubtreeProto {
-	height := s.layout.TileHeight(len(id) * 8)
+	const height = 8
```

Comment:
I'd be inclined to skip this here, not mention "with height %d" in the log message, and just inline it when setting the field below?

Reply:
Done.